test/drivers/test_suites/syscall_exit_suite/pread_x.cpp

```diff
@@ -0,0 +1,50 @@
+#include "../../event_class/event_class.h"
+
+#include <unistd.h>
+#include <sys/uio.h>
+
+#ifdef __NR_pread64
+
+TEST(SyscallExit, preadX_fail)
+{
+	auto evt_test = get_syscall_event_test(__NR_pread64, EXIT_EVENT);
+
+	evt_test->enable_capture();
+
+	/*=============================== TRIGGER SYSCALL ===========================*/
+
+	int32_t fd = -1;
+    size_t size = 13;
+    off_t pos = 17;
```

Comment:
minor: could you check leading whitespace vs. tabs.

Reply:
wops thanks!